setup.py

```diff
@@ -10,12 +11,16 @@
 with open('README.rst') as f:
     long_description = f.read()
 
+# Fixes a version conflict between numpy and scipy on python 3.9+
+scipy = "scipy" if sys.version_info < (3, 9) else "scipy>=1.7"
```

Comment:
can this say something like
`scipy = "scipy>=1.7" if sys.version_info >= (3, 9) and sys.version_info <= (3,10) else "scipy"`
instead?

It may age better

Reply:
Turns-out this was superfluous with the change from the outdated install method.